src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,34 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, $cardWidth);
+  justify-content: center;
+  gap: 48px;
+
+  max-width: 944px;
+  margin: 0 auto;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 200px;
+    height: 300px;
+
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    margin-bottom: 48px;
+  }
+  
+  @media (max-width: 1023.9px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (max-width: 767.9px) {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
+
+  @media (max-width: 487.9px) {
```

Comment:
You can use "min-width" and don't do fractional numbers, how and before the media queries